spec.html

```diff
@@ -6900,6 +6900,16 @@
         <emu-alg>
           1. Assert: IsPropertyKey(_P_) is *true*.
           1. Let _ownDesc_ be ? _O_.[[GetOwnProperty]](_P_).
+          1. Return OrdinarySetWithOwnDescriptor(_O_, _P_, _V_, _Receiver_, _ownDesc_).
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-ordinarysetwithowndescriptor" aoid="OrdinarySetWithOwnDescriptor">
```

Comment:
It'd be nice to include a comment here that this is used in WebIDL, but I think that's been shot down before.

Reply:
Yup: https://github.com/tc39/ecma262/pull/398#issuecomment-187724667. (I still think it would be useful...)